crypto/kzg/src/lib.rs

```diff
@@ -50,6 +59,7 @@ impl Kzg {
                 // Ref: https://notes.ethereum.org/@jtraglia/windowed_multiplications
                 8,
             )?,
+            context: PeerDASContext::default(),
```

Comment:
This is doing quite a few things, which we could also pull out (roughly):

```rust
let trusted_setup = TrustedSetup::default()
let prover_context = ProverContext::default()
let verifier_context = VerifierContext::default()
let num_threads = 16
PeerDASContext{trusted_setup, num_threads, prover_context, verifier_context} 
```
You can also initialize a trusted setup from the json string in the consensus-specs by doing `TrustedSetup::from_json(&str)` -- This is what `TrustedSetup::default()` is doing

Reply:
Closing as I've added the more explicit API